shared-components/tl-tournament-card.js

```diff
@@ -95,50 +37,133 @@ export class TournamentCard extends LitElement {
         return [
             css`
                 :host {
-                    /* Temporary style before the task is officially tackled */
-                    display: flex;
-                    flex-direction: column;
-
-                    align-items: center;
-                    justify-content: center;
+                /* Temporary style before the task is officially tackled */
+                display: flex;
+                flex-direction: column;
+                padding: 2rem;
+                min-height: 10rem;
+                height: auto !important;
```

Comment:
yeah. It could be bad. I just focused on making it like a design. The card height was already defined in another file and I should have removed it. I definitely agree with your way.